install_config/topics/full_rhv_inventory.adoc

```diff
@@ -0,0 +1,112 @@
+* Three Infrastructure Instances (Optional: Part of a negative affinity group)
+** Extra disks attached: 15GiB for Docker, 30 GiB for local volume storage, and 25GiB for registry Gluster
+* One or more Application Instances (Optional: Part of a negative affinity group)
+** Extra disks attached: 15GiB for Docker, 30 GiB for local volume storage
+* DNS entries for the routers and for the OpenShift web console
+
+[source,yaml]
+----
+[OSEv3:children]
+nodes
+masters
+etcd
+glusterfs_registry
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
+openshift_deployment_type=openshift-enterprise
+openshift_release='3.10'
+openshift_master_cluster_method=native
+debug_level=2
+openshift_debug_level="{{ debug_level }}"
+openshift_node_debug_level="{{ node_debug_level | default(debug_level, true) }}"
+openshift_enable_service_catalog=False
+
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+# Pod Networking
+os_sdn_network_plugin_name=redhat/openshift-ovs-networkpolicy
+
+# Registry
+openshift_hosted_registry_storage_kind=glusterfs
+
+# Authentication (example here creates one user, myuser with password changeme)
+openshift_master_identity_providers="[{'name': 'htpasswd_auth', 'login': 'True', 'challenge': 'True', 'kind': 'HTPasswdPasswordIdentityProvider', 'filename': '/etc/origin/master/htpasswd'}]"
+openshift_master_htpasswd_users={'myuser': '$apr1$zAhyA9Ko$rBxBOwAwwtRuuaw8OtCwH0'}
+
+# Red Hat Subscription Management
+# (vault_ variables are meant to be provided by Ansible Vault to protect passwords)
+rhsub_pool=Red Hat OpenShift Container Platform*
+rhsub_user="{{ vault_rhsub_user }}"
+rhsub_password="{{ vault_rhsub_password }}"
+
+# Docker and extra file system setup
+container_runtime_docker_storage_setup_device=/dev/vdb
+container_runtime_docker_storage_type=overlay2
+openshift_docker_use_system_container=False
+openshift_node_local_quota_per_fsgroup=512Mi
+openshift_use_system_containers=False
+
+[masters:vars]
+# Use the container_runtime section of the prerequisites playbook to set up extra file systems
+container_runtime_extra_storage="[{'device': '/dev/vdc', 'path': '/var/lib/origin/openshift.local.volumes', 'options': 'gquota', 'filesystem': 'xfs', 'format': 'True'}, {'device': '/dev/vdd', 'path': '/var/lib/etcd', 'hosts': 'masters', 'filesystem': 'xfs', 'format': 'True'}]"
+
+[nodes:vars]
+# Use the container_runtime section of the prerequisites playbook to set up extra file systems
+container_runtime_extra_storage="[{'device': '/dev/vdc', 'path': '/var/lib/origin/openshift.local.volumes', 'options': 'gquota', 'filesystem': 'xfs', 'format': 'True'}]"
```

Comment:
I'm not quite sure if above `container_runtime_extra_storage=...` has no problem, @jianlinliu could you help confirm?

Reply:
I have no idea about the option neither.

Reply:
https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example#L146 It allows for partitions to be created, mounted, and added to fstab to be used later on to segment off filesystems. It's important for the openshift_node_local_quota_per_fsgroup on all of the nodes. But its just an optional setting if we wanted we could remove it and make some inventory changes. It's just a nice to have